assets/prototype/application/hooks/useTimeZoneTime.ts

```diff
@@ -1,15 +1,15 @@
 import { zonedTimeToUtc, utcToZonedTime } from 'date-fns-tz';
-import useConfig from '../services/config/useConfig';
+import { configTypes, useConfig } from '../services/config/useConfig';
 
 const useTimeZoneTime = () => {
-	const { timezone } = useConfig();
+	const { timezone }: configTypes = useConfig();
```

Comment:
I am not sure if we need `configTypes` here because `ConfigDataProps` is already doing it's job:
![image](https://user-images.githubusercontent.com/1477453/71264221-462c0c80-234c-11ea-979a-74671c5a0139.png)

Reply:
ahh ok... cool... will change